puppet/data/common.yaml

```diff
@@ -0,0 +1,33 @@
+foreman_servicename: 'foreman.theforeman.org'
+foreman_url: 'https://%{lookup("foreman_servicename")}'
+puppet_servicename: 'puppet.theforeman.org'
+
+foreman::repo::repo: '3.1'
+foreman::config::apache::proxy_no_proxy_uris:
+  - /icons
+  - /server-status
```

Comment:
we override this, because you want `/server-status` to work?
if so, I think this deserves a comment a la "can be removed once we use `theforeman/foreman` whichever version" or something like that?

Reply:
Will add a note that we need at least 21.1.0